qmxgraph/widget.py

```diff
@@ -396,10 +396,15 @@ def _on_drop(self, event):
 
             # Refer to `mime.py` for docs about format
             version = parsed['version']
-            if version == 1:
-                x = event.pos().x()
-                y = event.pos().y()
+            if version not in (1, 2):
```

Comment:
> dropped from another application in that case we will will need the version info to properly handle the dropped data.

Oh sure, forgot that this was a possibility; it is a good practice anyway, was just wondering if we had a concrete use for it now.

> I actually have a feeling that drag&drop should be actually handled by the application. Any thoughts about it?

I see what you mean and I agree it makes sense. But I also think it makes sense for qmxgraph to support some drag & drop out of the box if most applications will want it anyway.

I guess let's leave it for now and see what other uses there are.